src/abstractdatatable/abstractdatatable.jl

```diff
@@ -380,7 +380,7 @@ function StatsBase.describe(io, dt::AbstractDataTable)
 end
 StatsBase.describe(nv::AbstractArray) = describe(STDOUT, nv)
 function StatsBase.describe{T<:Number}(io, nv::AbstractArray{T})
-    if all(_isnull, nv)
+    if all(isnull, nv)
```

Comment:
These `describe` functions on `AbstractArray` can be removed since they live in StatsBase now